examples/plotting/file/bar_intervals.py

```diff
@@ -11,8 +11,10 @@
 from bokeh.plotting import figure, show
 from bokeh.sampledata.sprint import sprint
 
-sprint.Year = sprint.Year.astype(str)
-group = sprint.groupby('Year')
+df = sprint.copy()  # since we are modifying sampledata
```

Comment:
In the long term it should be sampledata providing a fresh copy, because we are modifying the dataframe in sprint example as well, just without consequences.

Reply:
We could do that but I think running examples in their own subprocesses would also fix (and also be the best thing to do)

Reply:
In any case thanks for helping run this down